### DISCUSSION TITLE
gpx to mbd

### DISCUSSION THREAD

**Discussion by gsus24** [sk-zk/TruckLib#1] [Ideas]
Hi
nice to have somebody working on map generation.
How about generating roads with tracked gps data for 1:1 maps?
And for the next step detecting details like road type, railing, ... with recorded video
Best regards

**Comment by sk-zk** [comment]
Yeah, stuff like that would be the next step once I have this library in good shape. Intersections are tricky though: the game doesn't have procedurally generated intersections, like e.g. Cities: Skylines does, so you would either have to pick the closest existing prefab, or write a prefab generator which can create anything you need.
